### PR TITLE
Updated content based on feedback from the team

### PR DESCRIPTION
**Pull request by clagom**
After testing and discussing the first iteration of the process and convention the team gather feedback and worked on a second -more flexible- version of the practices. We will continue iterating on the process after testing this second version.
Reference doc: https://docs.google.com/document/d/16kl26jtOFfKL0ova2WEs66fQX6eJgG1yRITboKcrZs8/edit#

### CODE REVIEW

**Comment by amylily1011** [comment]
LGTM ;)

**Comment by clagom** [comment]
thanks @amylily1011 @squidsoup :)

File: workflow/UX-and-design-proposals.md [hunk 1]
```diff
@@ -1,154 +1,266 @@
...
+A - Discovery
+B - UX
+C - Visual Design
+D - Prototype
+E - User Testing
+F - Build
+G - QA
+
+
+## Resources
```
Comment (clagom) [inline workflow/UX-and-design-proposals.md:253]: Can we remove the Resources paragraph and links please

File: workflow/UX-and-design-proposals.md [hunk 2]
```diff
@@ -1,154 +1,266 @@
...
+
+-   <http://dschool-old.stanford.edu/wp-content/themes/dschool/method-cards/i-like-i-wish-what-if.pdf> 
+-   <http://www.andyhanselman.com/2013/05/03/i-like-i-wish-i-wonder-a-great-way-to-make-innovation-work-for-you/> 
+-   <https://www.ideo.com/blog/build-your-creative-confidence-i-like-i-wish> 
+-   <https://public-media.interaction-design.org/pdf/I-Like-I-Wish-What-If.pdf> 
+-   <https://medium.com/google-design/the-art-of-giving-feedback-80a6a4e2e843>
 
-## Resolving comments
+### Design critique
```
Comment (clagom) [inline workflow/UX-and-design-proposals.md:261]: Can we remove the Design critique paragraph and links please

**Comment by clagom** [comment]
Thanks @cristinadresch LGTM! After the checks are passed we can merge

**Pull request merged**